src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -87,6 +88,22 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
         }
     }
 
+    public async createChildImpl(context: ICreateChildImplContext): Promise<PostgresDatabaseTreeItem> {
+        const client: PostgreSQLManagementClient = createAzureClient(this.root, PostgreSQLManagementClient);
+        const fullID: string = nonNullProp(this, 'fullId');
+        const resourceGroup: string = azureUtils.getResourceGroupFromId(fullID);
+        const databaseName = await ext.ui.showInputBox({
+            placeHolder: "Database Name",
+            prompt: "Enter the name of the database",
+            validateInput: (name: string) => validateDatabaseName(name, client, this.name, resourceGroup)
+        });
+        context.showCreatingTreeItem(databaseName);
+        const database: Database = { name: databaseName };
+        await client.databases.createOrUpdate(resourceGroup, this.name, databaseName, database);
+        await this.refresh();
```

Comment:
This line will cause us to re-load all databases (which could be slow) and should be unnecessary